models/survey.py

```diff
@@ -143,8 +143,52 @@ def mark_deleted(self, survey_id):
     def get_results(self, survey_id):
         try:
             query = r.db(self.DB).table('Survey').get(survey_id).get_field('questions').map(
-                lambda doc: [doc, r.db(self.DB).table('QuestionResponse').filter({'question_id':doc}).get_field('response_data').coerce_to('array')]
+                lambda doc: [doc, r.db(self.DB).table('QuestionResponse').filter({'question_id': doc}).get_field('response_data').coerce_to('array')]
             ).coerce_to('object').run(self.conn)
             return query
-        except err:
+        except Exception as err:
+            logging.error(err)
+            return []
+
+    def get_formatted_results(self, survey_id):
+        results = self.get_results(survey_id)
+        try:
+            query = r.expr(results).coerce_to('array').map(
+                lambda question: r.db(self.DB).table('Question').get(question[0]).merge(r.expr({
+                    'results': question[1],
+                    'total_responses': question[1].count(),
+                    'pie_data': r.db(self.DB).table('Question').get(question[0]).do(
+                        lambda question_data: r.branch(
+                            (r.expr(question_data['response_format'] == Question().RESPONSE_MULTIPLE_CHOICE) | (question_data['response_format'] == Question().RESPONSE_TRUE_OR_FALSE)),
```

Comment:
Can you make these lines shorter? Python standard is 79 chars: https://www.python.org/dev/peps/pep-0008/#maximum-line-length